src/components/Select/Select.js

```diff
@@ -0,0 +1,33 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import ReactSelect from 'react-select'
+import ReactSelectAsync from 'react-select/async'
+
+import styles from './Select.module.scss'
+
+export const Select = ({ className, isAsync, ...rest }) => {
+  const props = {
+    className: `${className ? className : ''} ${styles.root}`,
```

Comment:
why do we need classname?

Reply:
I'm porting over previous version from monorepo and I'm not as worried about misuse of customizations, so I'll leave it.